build/images/base-windows/Dockerfile

```diff
@@ -61,3 +61,32 @@ COPY --from=windows-utility-base ["C:\\\\wins","C:\\\\wins"]
 COPY --from=windows-golang ["C:\\\\Program Files\\\\Go", "C:\\\\Program Files\\\\Go"]
 
 RUN setx /m PATH "C:\Program Files\7-Zip;C:\git\bin;C:\git\usr\bin;C:\mingw64\bin;C:\Program Files\Go\bin;%GOPATH\bin%;%PATH%"
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
```

Comment:
What are they used for? I didn't find their reference in the current code base.
Please add description in the commands' comments.

Reply:
This command is used to download and install vc, it is required be installed in base image to run ovs processes, there is a comment just above this command, "install vc"

Reply:
1. "Install VC" can be easily understood from the code itself, "it's required to be installed in base image to run ovs processes" can't be told from the code, which is more worth a comment than "Install VC".
2. I doubt why we need to install VC if it's used to run OVS processes, because base-windows is an intermediate image used to build dependency only, not the eventual image which runs antrea-agent and OVS. How was it tested in a way it would be eventually consumed?

cc @antoninbas @XinShuYang @wenyingd

Reply:
> 1. "Install VC" can be easily understood from the code itself, "it's required to be installed in base image to run ovs processes" can't be told from the code, which is more worth a comment than "Install VC".

We can elaborate the comment



> 2\. I doubt why we need to install VC if it's used to run OVS processes, because base-windows is an intermediate image used to build dependency only, not the eventual image which runs antrea-agent and OVS. How was it tested in a way it would be eventually consumed?

antrea-agent and ovs eventually uses base-windows(antrea-windows image uses base-windows to build ,       FROM antrea/base-windows:${WIN_BUILD_TAG} as antrea-build-windows)
 so ovs and other dependencies are built in base-windows.

It is well tested via the PR https://github.com/antrea-io/antrea/pull/5052

Reply:
`antrea-windows` image's base image is `mcr.microsoft.com/powershell:lts-nanoserver-${NANOSERVER_VERSION}`, instead of `base-windows`: https://github.com/antrea-io/antrea/blob/2b864691893ad23a88cfcd0bfd03cb2b1c1ca8eb/build/images/Dockerfile.build.windows#L30

You can see other binaries are copied from `base-windows` explicitly then they will be available in `antrea-windows`. The OVS bindaries are downloaded to `base-windows` image but not even copied to `antrea-windows. Then how #5052 work? Is there a suceessful build I can look into?